blocks/form/file.js

```diff
@@ -0,0 +1,52 @@
+function attachFiles(wrapper, fileList, fileTemplate) {
+    for(let i = 0; i < fileList.length; i++) {
+        const file = fileList[i];
```

Comment:
you could check allowed file extensions and mime types as a basic spoofing protection in the following line:

```
        if (!isFileAllowed(file)) {
            console.error(`File extension or mime type not allowed: ${file.name}`);
            continue;
        }
```

the function could look like this:
```
function isFileAllowed(file) {
  const allowedExtensions = ['jpg', 'jpeg', 'png', 'gif', 'pdf']; // Customize this list based on your needs
  const allowedMimeTypes = ['image/jpeg', 'image/png', 'image/gif', 'application/pdf']; // Customize this list based on your needs

  const extension = file.name.split('.').pop().toLowerCase();
  const mimeType = file.type; // The MIME type can be accessed directly from the file object in the browser

  return allowedExtensions.includes(extension) && allowedMimeTypes.includes(mimeType);
}

```